packages/server/src/middleware/nuxt.js

```diff
@@ -74,7 +74,7 @@ export default ({ options, nuxt, renderRoute, resources }) => async function nux
     res.setHeader('Accept-Ranges', 'none') // #3870
     res.setHeader('Content-Length', Buffer.byteLength(html))
     res.end(html, 'utf8')
-    nuxt.callHook('render:routeDone', url, result, context)
+    await nuxt.callHook('render:routeDone', url, result, context)
```

Comment:
This creates extra promise object on each request. Is it necessary for other than tests?

Reply:
`callHook` will return promise anyway, if we don't wait here, there will have unhandled promise.